cirq-core/cirq/testing/gate_features.py

```diff
@@ -17,6 +17,13 @@
 from cirq.ops import raw_types
 
 
+class SingleQubitGate(raw_types.Gate):
```

Comment:
This seems to be used inconsistently in tests - there are some tests which switched to use it, but others are now `cirq.Gate` with `_num_qubits_` definitions. What was the deciding factor for this?

Reply:
> but others are now cirq.Gate with _num_qubits_ definitions

I started with this before i realized i could use the `cirq.SingleQubitGate` in `testing`. I thought i got every one of these but seems like there are some left. Will revert the ones that use `cirq.Gate` in the next commit.

Reply:
Just got rid of the inconsistencies.